src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -242,8 +242,8 @@ export class RelayFeeCalculator {
   ): Promise<BigNumber> {
     if (toBN(amountToRelay).eq(bnZero)) return MAX_BIG_INT;
 
-    const { inputToken } = deposit;
-    const token = getTokenInformationFromAddress(inputToken, tokenMapping);
+    const { inputToken, originChainId } = deposit;
+    const token = getTokenInfo(inputToken, originChainId, tokenMapping);
```

Comment:
This might not resolve as expected if the originChainId == hubChainID if different l2 tokens have the same l1 token but have different decimals

Reply:
That's correct. Fixed here https://github.com/across-protocol/sdk/pull/999/commits/05f60fd9647ead5146db783fd310782170a13189